api/codes.go

```diff
@@ -35,10 +55,17 @@ func (c *CodeListAPI) getCodes(w http.ResponseWriter, r *http.Request) {
 		codes.Items[i] = item
 	}
 
-	count := len(codes.Items)
+	sort.Slice(codes.Items, func(i, j int) bool {
+		return codes.Items[i].ID < codes.Items[j].ID
+	})
+
+	slicedResults := codesSlice(codes.Items, offset, limit)
```

Comment:
The `slicedResults` are not being put into the response model, so all results are being returned